src/game/emutest.c

```diff
@@ -23,7 +23,7 @@ extern void __osPiGetAccess(void);
 extern void __osPiRelAccess(void);
 
 enum Emulator gEmulator = EMU_CONSOLE;
-u8 gSupportsLibpl = FALSE;
```

Comment:
I assume this is just an int rather than an enum because it wouldn't compile without the submodule?

Reply:
Yeah. C doesn't care about type safety of enums, so it still works as an int.